modules/Bio/EnsEMBL/IO/Parser/Genbank.pm

```diff
@@ -282,6 +285,36 @@ sub get_sequence_version {
     return $self->{'record'}->{'_version'};
 }
 
+=head2 get_dbsource
+
+    Description: Return the dbsource from the GenBank file
+    Returntype : String
+
+=cut
+
+sub get_dbsource {
+    my $self = shift;
+
+    return $self->{'record'}->{'_raw_dbsource'};
+}
+
+=head2 get_dbsource_acc
+
+    Description: Return the dbsource accession from the GenBank file
+    Returntype : String
+
+=cut
+
+sub get_dbsource_acc {
+    my $self = shift;
+
+    return unless $self->{'record'}->{'_raw_dbsource'};
+
+    my ($acc) = $self->{'record'}->{'_raw_dbsource'} =~ /REFSEQ: accession (\S+)/;
```

Comment:
Are you guaranteed that these don't line-wrap? I can't remember how far indented they are.